feeds/src/index.tsx

```diff
@@ -3,12 +3,13 @@ import ReactDOM from 'react-dom'
 import { Provider as ReduxProvider } from 'react-redux'
 import * as serviceWorker from './serviceWorker'
 import App from './App'
+import { Config } from './config'
 import createStore from './state/createStore'
 import { PersistGate } from 'redux-persist/integration/react'
 import ReactGA from 'react-ga'
 import './theme.css'
 
-ReactGA.initialize(process.env['REACT_APP_GA_ID'] || '')
+ReactGA.initialize(Config.gaId())
```

Comment:
I'm having following TS issue:
Looks like there is a type conflict in `initialize`
```
TypeScript error in /Users/bartosz/localhost/chainlink/feeds/src/index.tsx(12,20):
No overload matches this call.
  Overload 1 of 2, '(trackingCode: string, options?: InitializeOptions | undefined): void', gave the following error.
    Argument of type 'string | undefined' is not assignable to parameter of type 'string'.
      Type 'undefined' is not assignable to type 'string'.
  Overload 2 of 2, '(trackers: Tracker[], options?: InitializeOptions | undefined): void', gave the following error.
    Argument of type 'string | undefined' is not assignable to parameter of type 'Tracker[]'.
      Type 'undefined' is not assignable to type 'Tracker[]'.  TS2769

    10 | import './theme.css'
    11 | 
  > 12 | ReactGA.initialize(Config.gaId())
```

Reply:
@hellobart fixed